northd/northd.c

```diff
@@ -12804,12 +12804,18 @@ build_ND_RA_flows_for_lrouter_port(
         ds_clear(match);
         ds_put_format(match, "inport == %s && ip6.dst == ff02::2 && "
                       "nd_ra && "REGBIT_ND_RA_OPTS_RESULT, op->json_key);
-
         char ip6_str[INET6_ADDRSTRLEN + 1];
-        struct in6_addr lla;
-        in6_generate_lla(op->lrp_networks.ea, &lla);
-        memset(ip6_str, 0, sizeof(ip6_str));
-        ipv6_string_mapped(ip6_str, &lla);
+        if (op->peer->proxy_arp_addrs.n_ipv6_addrs) {
+            VLOG_WARN("deleteme, config RA src ip with arp_proxy");
```

Comment:
Do we need this VLOG_WARN ?  I guess you need to clean this up.
same below in else too